include/mlir/ExecutionEngine/OptUtils.h

```diff
@@ -41,17 +42,23 @@ void initializeLLVMPasses();
 
 /// Create a module transformer function for MLIR ExecutionEngine that runs
 /// LLVM IR passes corresponding to the given speed and size optimization
-/// levels (e.g. -O2 or -Os).
+/// levels (e.g. -O2 or -Os). If not null, `targetMachine` is used to
+/// initialize passes that provide target-specific information to the LLVM
+/// optimizer. `targetMachine` must outlive the returned std::function.
 std::function<llvm::Error(llvm::Module *)>
-makeOptimizingTransformer(unsigned optLevel, unsigned sizeLevel);
+makeOptimizingTransformer(unsigned optLevel, unsigned sizeLevel,
```

Comment:
Please update the call sites in examples/toy/Ch5/toyc.cpp and the likes

Reply:
see https://source.cloud.google.com/results/invocations/d9d8a397-5ac0-4333-82e2-a0706b00a1df/log

Reply:
Sorry, I had examples disabled. Done. Thanks!

Reply:
Thanks!